RealmSwift-swift2.0/Tests/RealmCollectionTypeTests.swift

```diff
@@ -101,7 +101,9 @@ class RealmCollectionTypeTests: TestCase {
     }
 
     func testDescription() {
+        // swiftlint:disable line_length
```

Comment:
After seeing a few of these, it makes me think there could be value in having something an annotation that applies only to the immediately following item (line, function, class, etc. depending on context), to avoid having to pair these in cases where they only apply to one item.